reactor-netty-core/src/main/java/reactor/netty/transport/ByteBufAllocatorMetrics.java

```diff
@@ -65,59 +55,49 @@ private ByteBufAllocatorMetrics() {
 
 	void registerMetrics(String allocType, ByteBufAllocatorMetric metrics, ByteBufAllocator alloc) {
 		MapUtils.computeIfAbsent(cache, metrics.hashCode() + "", key -> {
-			String[] tags = new String[] {ID, key, TYPE, allocType};
+			Tags tags = Tags.of(ID.getKey(), key, TYPE.getKey(), allocType);
 
-			Gauge.builder(BYTE_BUF_ALLOCATOR_PREFIX + USED_HEAP_MEMORY, metrics, ByteBufAllocatorMetric::usedHeapMemory)
-			     .description(USED_HEAP_MEMORY_DESCRIPTION)
+			Gauge.builder(USED_HEAP_MEMORY.getName(), metrics, ByteBufAllocatorMetric::usedHeapMemory)
```

Comment:
ah I see, it's the reverse: the builder uses the enum's `getName()` unless it needs something more dynamic (in which case the enum's `getName()` doesn't really matter?)

Reply:
yes, when it is dynamic it doesn't matter